Sources/App/Services/APIConnect+Services.swift

```diff
@@ -62,6 +62,13 @@ extension APIConnect where From == Slack.Request {
 // MARK: Slack.Request -> CircleCi.JobRequest
 extension APIConnect where From == Slack.Request, To == CircleCi.JobRequest {
     static func run(_ from: Slack.Request, _ context: Context) -> IO<Slack.Response?> {
+        if Environment.get(key: "debugMode") == "true" {
```

Comment:
There's a "global" run(...) if I'm not mistaken, which called by every request for response. Can we move request and response printing logic to that place?

Reply:
I didn't find better place for incoming logs. Can we talk about it and check the project?